test/v1/support/DownstreamResponseMappingSupportSpec.scala

```diff
@@ -68,6 +69,14 @@ class DownstreamResponseMappingSupportSpec extends UnitSpec {
       }
     }
 
+    "ifs returns UNMATCHED_STUB_ERROR" must {
```

Comment:
Can we replace word `ifs` with `downstream` as it is only our stub that will return this